compass/ocean/tests/global_ocean/init/initial_state.py

```diff
@@ -160,6 +170,36 @@ def run(self):
 
         run_model(self)
 
+        if self.with_inactive_top_cells:
+
+            logger.info("   * Updating minLevelCell for inactive top cells")
+
+            in_filename = 'initial_state.nc'
+            out_filename = in_filename
+
+            with xarray.open_dataset(in_filename) as ds:
+
```

Comment:
In my testing, `xarray` is getting confused in the following.  Because of its "lazy" loading of files, it doesn't deal well with a file being read in and written out to the same file name.  The best way around this is to call `ds.load()` immediately after opening the file to make sure the file is read into memory.  After doing that, things work as expected.
```suggestion
            with xarray.open_dataset(in_filename) as ds:

                ds.load()
```